website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1649,6 +1656,34 @@ var Range = ace.require('ace/range').Range;
                 buttonRow.appendChild(button);
                 return button;
             };
+            var makeHelpButton = function(id,title,XShift){
+              var button = document.createElement("li");
+              //Based off the search button classes
+              //<button type=button class="btn osf-search-btn" data-toggle="modal" data-target="#search-help-modal"><i class="fa fa-question fa-lg"></i></button>
+              //$(".dropdown-toggle").attr("data-toggle", "dropdown");
+              button.className = "wmd-button";
+              // change those attributes
+              button.setAttribute('data-toggle','modal');
+              button.setAttribute('data-target','#wiki-help-modal');
+
+              togg = button.getAttribute('data-toggle');

```

Comment:
Whole words are a good idea with variables. Partially because there's no character limit, so if it helps readability and isn't excessive, then yay! Also partially because people using IDEs will get spell check squiggles if it's not made up of real words, so no need for extra squiggles.